Makefile.dep

```diff
@@ -59,6 +59,10 @@ ifneq (,$(filter gnrc_conn_udp,$(USEMODULE)))
   USEMODULE += gnrc_udp
 endif
 
+ifneq (,$(filter gnrc_netreg_extra,$(USEMODULE)))
```

Comment:
I can fix it up to devide it into two modules, but the complexity is growing in both instances.